ru/lessons/specifics/debugging.md

```diff
@@ -0,0 +1,191 @@
+---
+version: 1.0.0
+title: Отладка
+redirect_from:
```

Comment:
I suppose that we don't need redirect for the Russian translation. Earlier the original English lessons didn't have their own language folder. It was added after a [huge rework](https://github.com/elixirschool/elixirschool/pull/1112) (thanks to @eksperimental) and since that moment a redirect had to be introduced in order to keep the old links alive.
So I suggest to remove lines 4-5.